stacks/dovetail.prx.org.yml

```diff
@@ -289,7 +354,7 @@ Resources:
       DeploymentConfiguration:
         MaximumPercent: 200
         MinimumHealthyPercent: 50
-      DesiredCount: !GetAtt DovetailServiceDesiredCountRolloverResource.DesiredCount
+      DesiredCount: 10
```

Comment:
Per the exchange-worker autoscaling PR, you no longer have to worry about this blowing away any autoscaling changes to the desired count.

Reply:
Want to remove that rollover custom resource in this same PR?

Reply:
JK you did